main.tf

```
@@ -4,16 +4,24 @@
 # We might add more than one line for additional information
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-resource "google_logging_project_sink" "sink" {
-  count = var.module_enabled ? 1 : 0
+locals {
+  create_project_sink      = (var.project != null) || (var.folder == null && var.organization == null)
```

we always create the project sink even with `null` project? 

```suggestion
  create_project_sink      = var.folder == null && var.organization == null
```